src/stubs/LocalizedVagrantfile

```diff
@@ -0,0 +1,23 @@
+require 'json'
+require 'yaml'
+
+VAGRANTFILE_API_VERSION = "2"
+confDir = $confDir ||= File.expand_path("~/.composer/vendor/laravel/homestead")
```

Comment:
This PR assumes the user has homestead installed globally and wants to install into $this project folder they are in. 

If we want to build the same / similar thing into the Laravel command,  think that's viable but different scenario. We talked about going either way with it.